strings/traumas.json

```diff
@@ -1,153 +1,111 @@
+        "If I had a tail... @pick(qotsareference)",
+        "I'm late! I'm late! For a very important date!",
+        "I think I brained my damage...",
+        "Take it easy!",
+        "Ow!!",
+        "Too many faces...",
+        "I want... a @pick(muffin) muffin...",
+        "Poppy |seed!!|",
+        "I wanna dig a hole...",
+        "Space is so cool...",
+        "Rats...",
+        "Why is the room shaking?",
+        "Welcome to @pick(barname)...!",
+        "Call the shuttle!",
+        "Medical is scary...",
+        "I don't know... That's scary...",
+        "You're a big guy...",
+        "Cheese...",
+        "I need thoughts and prayers...",
+        "Get away from me!",
+        "The air tastes all itchy...",
+        "@pick(semicolon) Help me!",
+        "What does that mean...",
+        "Who the hell do you think I am...?",
+        "Oh hi, @pick(name)!",
+        "@pick(semicolon) Ai... I want a @pick(muffin) muffin...",
+        "This is my dorm! It was made for me!",
+        "Who the hell do you think you are?",
+        "Get out...",
+        "Moth party!",
+        "I love moths...",
+        "Ugh... My head hurty...",
+        "Pew pew! Pshoom!",
+        "For you...",
+        "Pow pow! Dakka dakka dakka!",
+        "Braaains...",
+        "I wonder how you make crème brûlée...",
+        "No... No needles!",
+        "@pick(semicolon)Tajaran has wares... if you have coin...",
+        "@pick(semicolon)I've got balls of steel!",
+        "I can take a whole deathsquad! ...In a fight!",
+        "Happy birthday to you...~",
+        "@pick(semicolon) This reminds me of 2484...",
+        "Cubic symmetry four day station...",
+        "Hehe... Bananas...",
+        "@pick(semicolon) N-NYAAAAAA~",
+        "Don't ever touch me!",
+        "I need credits... About three fifty...",
+        "The singularity is coming!!"
     ],
 
-    "mutations": [
-        "telikesis",
-        "halk",
-        "eppilapse",
-        "kamelien",
-        "eksrey",
-        "glowey skin",
-        "fungal tb",
-        "stun gloves"
+    "powers": [
+        "fly",
+        "do anything",
+        "breathe fire",
+        "disappear",
+        "breathe plasma",
+        "eat a whole pizza all by myself"
     ],
 
-    "george": [
-        "joerge",
-        "george",
-        "gorge",
-        "gdoruge"
+    "qotsareference": [
+        "I'd own... The place...",
+        "I'd swat... The flies..."
     ],
-
-    "mellens": [
-        "mellens",
-        "melons",
-        "mwrlins"
-    ],
-    "random_gibberish": [
-        "g",
-        "squid",
-        "r",
-        "carbon dioxide"
+	
+	"muffin": [
+        "banana nut",
+        "poppy seed",
+		"blueberry",
+		"chocolate chip"
     ],
 
-    "y_replacements": [
-        "y",
-        "i",
-        "e"
+    "barname": [
+        "The Maltese Falcon",
+        "The Maid Cafe",
+        "The Loose Goose",
+        "Vlad's Salad Bar",
+        "The Wretched Hive"
```

Comment:
the implication that someone with brain damage could say any of these with no fault in speech upsets me